smart-contracts/osmosis/contracts/cl-vault/src/contract.rs

```diff
@@ -292,8 +290,6 @@ pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, Co
     OLD_POSITION.remove(deps.storage);
     #[allow(deprecated)]
     CURRENT_BALANCE.remove(deps.storage);
-    #[allow(deprecated)]
-    CURRENT_SWAP.remove(deps.storage);
```

Comment:
Afaik this remove was not migrated yet to mainnet contracts, if so we should inline the remove so we don't orphan state items

Reply:
I don't get what you refer to with "orphaning" items, what is that and what is the real problem with removing this?